backend/api/go_client/visualization.pb.go

```diff
@@ -43,20 +43,29 @@ type Visualization_Type int32
 
 const (
 	Visualization_ROC_CURVE Visualization_Type = 0
+	Visualization_TFDV      Visualization_Type = 1
+	Visualization_TFMA      Visualization_Type = 2
+	Visualization_TABLE     Visualization_Type = 3
 )
 
 var Visualization_Type_name = map[int32]string{
```

Comment:
Maybe in future we can find a way to have a dynamic list of visualizations without hard-coding them in the compiled back-end code.

Frontend could scan the `components/visualizers` directory and list all visualizers from there.

Reply:
This would be ideal if we continue to add visualizations at a rapid rate. Perhaps that can be investigated if that is the case for the future. It would reduce PR size and developer requirements for introducing new visualizations.